resources/migrations/001_update_migrations.yaml

```diff
@@ -9440,11 +9440,11 @@ databaseChangeLog:
                   constraints:
                     nullable: true
               - column:
-                  remarks: Random NanoID tag for unique identity.
                   name: entity_id
                   type: char(21)
+                  remarks: Random NanoID tag for unique identity.
                   constraints:
-                    nullable: false
```

Comment:
apparently serdes require this to be nullable... see the `metabase-enterprise.serialization.v2.entity-ids-testentity-ids-are-nullable` test